packages/frontend/src/components/common/ResourceList/ResourceActionMenu.tsx

```diff
@@ -76,23 +77,27 @@ const ResourceListActionMenu: FC<Props> = ({ item, spaces, url, onAction }) => {
                             });
                         }}
                     />
-                    <MenuItem2
-                        role="menuitem"
-                        icon="pin"
-                        text={
-                            isPinned ? 'Unpin from homepage' : 'Pin to homepage'
-                        }
-                        onClick={(e) => {
-                            e.preventDefault();
-                            e.stopPropagation();
+                    <Can I="manage" a="SavedChart">
```

Comment:
you need more specific check here based on org and project

should be like this:

```tsx
<Can I="manage" this={subject('SavedChart', { organizationUuid, projectUuid })}>
   ...
```